src/tx/applyTx/checkSpendCond.js

```diff
@@ -246,11 +246,9 @@ module.exports = async (state, tx, bridgeState, nodeConfig = {}) => {
     let allowance;
 
     if (!spendingIsOwner) {
-      if (unspent.address !== input.signer) {
+      if (input.signer && unspent.address !== input.signer) {
```

Comment:
if an input is unsigned, it might be an ERC1948 and it's data will be updated through `writeDataByReceipt()`